mac/README.md

```diff
@@ -6,47 +6,30 @@ Xcode 8.3.3 (it might also work to use an older version)
 ## Keyman for macOS Development
 Keyman for macOS can be built from a command line (preferred) or Xcode.
 
-### Compiling from Command line
+### Compiling from Command Line
 To build Keyman for macOS, do the following:
-1. Open a Terminal window.
-2. Change to please run the build.sh build script within this folder. Run
-with the -help switch to see all options.
-You will need to either run with the command-line argument -no-codesign or obtain the
-necessary security permissions and certificate data as part of our core development team
-to complete the build for the final app.
-To build using Xcode, you will need to build KeymanEngine4Mac first and then build
-Keyman4MacIM.
+1. Open a terminal
+2. cd to **keyman/mac**. **build.sh** must be run in the directory containing the script.
+3. Build using `./build.sh -no-codesign`. Run `./build.sh -help` to see all options.
 
-Keyman4MacIM is an input method, and as such is installed in the Input Methods folder.
-The easiest way to install it locally for testing is to specify "-deploy local" on the
-command line. See https://help.keyman.com/products/macosx/start_download-install_keyman.php
-for complete instructions on setting up Keyman for the first time and installing keyboards.
-
-The Keyman4Mac project builds a “test-bed” app that can be used to test keyboards without
```

Comment:
I added back a little about Keyman4Mac. I still left the examples out though because instructions on how to build a release version aren't necessary for external contributors and I found that looking `./build.sh -help` is enough to figure it out.